static/local/blackwomenssuffrage/key-figures.md

```diff
@@ -1,83 +1,199 @@
 of suffrage for Black women. In 1891, helped found the interracial Provident Hospital to create opportunities for Black doctors and nurses, as well as its associated nursing school for Black women. In order to advocate for more resources for Black women, including access to education, childcare, employment assistance and banks, Williams helped to found the [National League of Colored Women](/search?q=National+League+of+Colored+Women) in 1893.Three years later, she was active in the merging of this group with others to form the [National Association of Colored Women](/search?q=National+Association+of+Colored+Women) (NACW). During the planning of the 1893 Columbian Exposition in Chicago, she worked to have African Americans included in the planning process and to ensure representation in the exhibits. She also gave a speech entitled "The Intellectual Progress and Present Status of the Colored Women of the United States since the Emancipation Proclamation," to the Departmental Congress of the National Association of Loyal Women of American Liberty at the [World's Congress of Representative Women](/search?q=Worlds+Congress+of+Representative+Women), in which she described the obstacles facing Black women searching for employment, arguing that ability and talent counted "for nothing." Several months later, she gave a speech to the [World's Parliament of Religions](/search?q=Worlds+Parliament+of+Religions) about racial equality within religious organizations, and the important role that organized religion could play in building a better society. With these two speeches, she gained a national reputation and became a sought after speaker. She was nominated for membership in the exclusive [Chicago Women's Club](/search?q=Chicago+Womens+Club), setting off a long debate within the all-white club about granting membership to a Black woman and prompting several resignations when she was accepted in 1895. Her history of the Black women's club movement was published in 1902, which, she wrote, had grown to 400 clubs with a total membership of more than 150,000. In 1905, Williams and her husband, along with several other families, founded the Frederick Douglass Center settlement project. She also served as a reporter for Woman's Era, a newspaper published by fellow suffragist Josephine St. Pierre Ruffin and her daughter, to provide news reports for and by Black women, while also writing for the Chicago Record-Herald and the New York Age. After the death of white suffragist leader Susan B. Anthony in 1907, Williams was the only Black woman asked to speak at Anthony's funeral.  In 1909, Williams was active in the creation of the [National Association for the Advancement of Colored People](/search?q=National+Association+for+the+Advancement+of+Colored+People) (NAACP).  In 1924, after the death of her husband, she became the first Black woman to sit on the Chicago Library Board. Williams returned to Brockport, New York, in 1926 and died there in 1944 at the age of 89.
```

Comment:
You can add a link to Frances "Fannie" Barrier Williams `/search?q=Fannie+Barrier+Williams`